src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageReadmeControl.xaml

```diff
@@ -28,8 +30,13 @@
       x:Uid="descriptionMarkdownPreview"
       ClipToBounds="True"
       Focusable="False"
-      Visibility="{Binding Path=ErrorLoadingReadme, Mode=OneWay, Converter={StaticResource NegatedBooleanToVisibilityConverter}}"
+      Visibility="{Binding Path=IsReadmeReady, Mode=OneWay, Converter={StaticResource BooleanToVisibilityConverter}}"
```

Comment:
There are 4 states for the Readme operations and results:

1. The Readme has not been loaded yet
2. The Readme is currently loading
3. The Readme has finished loading and it updated successfully
4. The Readme has finished loading and it failed

During state 2, the Loading indicator should be shown. Based on this, I don't think it's doing the proper thing to determine if the loading indicator should be based on the computed property IsReadmeReady, which is in turn based on ErrorLoadingReadme. ErrorLoadingReadme will only be set after the loading is complete and should be cleared (set to false) at the beginning of the next loading (if we were to implement a refresh or retry, for example).

I'm not seeing the need for the IsReadmeReady property or its use here but please help me understand the scenario where it's needed. Note, I have feedback in another comment about how ErrorLoadingReadme might not be appropriate for the ViewModel considering its use is to determine if the markdown View component failed during updating.